assets/js/modules/analytics-4/components/common/UseSnippetSwitch.js

```diff
@@ -40,10 +42,10 @@ export default function UseSnippetSwitch() {
 	const onChange = useCallback( () => {
 		setUseSnippet( ! useSnippet );
 		trackEvent(
-			'analytics_setup',
-			useSnippet ? 'analytics4_tag_enabled' : 'analytics4_tag_disabled'
+			`${ viewContext }_analytics`,
+			useSnippet ? 'disable_ga4_tag' : 'enable_ga4_tag'
```

Comment:
ATTN reviewer: if you're wondering why these are flipped, from my testing they've always been wrong, see:
/wp-admin/admin.php?page=googlesitekit-settings#/connected-services/analytics/edit

Also since we're setting the useSnippet to its inverse (which updates the ui) then it goes that the behaviour should replicate.

Reply:
Perhaps we can do something like the following to make it less confusing?

```js
const onChange = useCallback( () => {
    const newUseSnippet = ! useSnippet;
    setUseSnippet( newUseSnippet );
    trackEvent( `...`, newUseSnippet ? 'enable_ga4_tag' : 'disable_ga4_tag' );
}, [ ... ] );
```

Reply:
@ivankruchkoff see Felix's comment regarding the event name and label change: https://github.com/google/site-kit-wp/issues/4216#issuecomment-963483268